libraries/botbuilder-adapter-alexa/src/alexaAdapter.ts

```diff
@@ -0,0 +1,150 @@
+import {
+    BotAdapter,
+    ConversationReference,
+    TurnContext,
+    Activity,
+    ResourceResponse,
+    WebRequest,
+    WebResponse,
+    ActivityTypes
+} from 'botbuilder';
+import { RequestEnvelope, IntentRequest, Session, interfaces, ResponseEnvelope, SessionEndedReason, SessionEndedRequest } from 'ask-sdk-model';
+import { getRequestType, getLocale, getUserId, createAskSdkError } from 'ask-sdk-core';
+import { retrieveBody } from '@botbuildercommunity/adapter-twitter';
```

Comment:
Future note: We have `retrieveBody` in three packages now. Might be time to make a `@botbuildercommunity/util` or something.

Reply:
> How closely does this align with @iMicknl’s WIP PR for an Alexa adapter? Can they be merged?

I will have a look. This one looks more detailed than mine, so will have a look if I did some things different and what we should merge. I paused mine since @garypretty is releasing a new version of the .NET adapter and I wanted to base it on the new version. 

Thanks for your contribution @trashcanmonster8.

Reply:
I am not a fan of referring other packages, however having the code duplication is also not great.. Best would be to have a util package indeed or to have it in the official botframework adapter, which can be referenced.